src/Helper/SplitStreet.php

```diff
@@ -57,9 +93,25 @@ class SplitStreet
      */
     public static function splitStreet(string $fullStreet, string $local, string $destination): FullStreet
     {
-        $translateBoxSeparator = str_ireplace(self::BOX_TRANSLATION_POSSIBILITIES, ' ' . self::BOX_NL, $fullStreet);
-        $fullStreet            = trim(preg_replace('/(\r\n)|\n|\r/', ' ', $translateBoxSeparator));
-        $regex                 = self::getRegexByCountry($local, $destination);
+        // Replace house number suffix by an abbreviation, only possible for the Netherlands
+        if ($destination === AbstractConsignment::CC_NL) {
+            foreach (self::NUMBER_SUFFIX_ABBREVIATION as $from => $to) {
+                $fullStreet = preg_replace("/(\d.*-?)[\s]$from/", '$1' . $to, $fullStreet);
```

Comment:
Dit moet eigenlijk `"/(\d.*-?)[\s]$from/i"` zijn, anders pakt hij niet de toevoeging als er een hoofdletter aanwezig is.